tests/base/managers/entity.py

```diff
@@ -591,3 +591,56 @@ def test_mgr_entity_search_translation(entity_test_search):
     assert search('first translation') == [entities[0]]
     assert search('second translation') == [entities[1]]
     assert search('third translation') == [entities[2]]
+
+
+@pytest.mark.django_db
+def test_lookup_collation(resource0, locale0, translation_factory):
+    """
+    Filter translations according to collation.
+    """
+    entity = Entity.objects.create(
+        resource=resource0,
+        string="string")
+
+    batch_kwargs = sum(
```

Comment:
i think the sum is unnecessary here and complicates the code a little - why not just use a list of dicts ?

Reply:
you could also do this...

```
batch_kwargs = [dict(entity=entity, locale=locale0, string=s) for s in [u'string 1', u'string 2']]
```